lib/reopen_issues_by_mail/mail_handler_patch.rb

```diff
@@ -14,7 +14,9 @@ def receive_issue_reply_with_reopen_issues_by_mail(issue_id, from_journal=nil)
         issue = Issue.find_by_id(issue_id)
         return unless issue
         if issue.closed?
-          status_id = IssueStatus.default.id
+          # 1 = new
+          status_id = 1
+          #status_id = IssueStatus.default.id
```

Comment:
I have not touched Redmine (or even Rails) in years and years, but this looks a bit hacky to me. What's the problem this tries to address? That `IssueStatus.default` can be nil in some system? If that's the case, I think this would be better:

```ruby
# If there is no default status, force it to 1 (which means "new")
status_id = IssueStatus.default ? IssueStatus.default.id : 1
```

Having into account this is Rails, I guess this would work as well:

```ruby
# If there is no default status, force it to 1 (which means "new")
status_id = IssueStatus.default.try(:id) || 1
```

And with Ruby > 2.3.0 you could also do

```ruby
# If there is no default status, force it to 1 (which means "new")
status_id = IssueStatus.default&.id || 1
```

Take into account my three alternatives do the same - introducing your fix of enforcing a 1 without destroying the old logic.